query/src/servers/http/v1/query/execute_state.rs

```diff
@@ -71,24 +73,47 @@ impl ExecuteState {
 
 use ExecuteState::*;
 
-pub(crate) type ExecuteStateRef = Arc<RwLock<ExecuteStateWrapper>>;
+pub(crate) type ExecutorRef = Arc<RwLock<Executor>>;
```

Comment:
It would be better to introduce a new struct wrapping Executeor with RwLock here, 

struct ExecutorState {Arc<RwLock<Executor>>} and implement ExecutorState , like https://github.com/datafuselabs/databend/pull/2998

Reply:
We could do it in another PR

Reply:
thank you, I will rethink it later.